samples/BindingTest/App.xaml.cs

```diff
@@ -13,30 +13,20 @@ public class App : Application
         public App()
         {
             RegisterServices();
-            InitializeSubsystems((int)Environment.OSVersion.Platform);
-            InitializeComponent();
-            InitializeLogging();
-        }
-
-        public static void AttachDevTools(Window window)
-        {
-            DevTools.Attach(window);
         }
 
         private static void Main()
         {
-            var app = new App();
-            var window = new MainWindow();
-            window.Show();
-            app.Run(window);
-        }
+			InitializeLogging();
 
-        private void InitializeComponent()
-        {
-            PerspexXamlLoader.Load(this);
+			new App()
+				.UseWin32Subsystem()
+				.UseDirect2D()
+				.LoadFromXaml()
+				.RunWithMainWindow<MainWindow>();
         }
 
-        private void InitializeLogging()
+		private static void InitializeLogging()
```

Comment:
Whitespace looks messed up here and elsewhere. Are you using a tab spacing of 3 or something? We use spaces rather than tabs and a spacing of 4.